manifests/Jibauni.yaml

```
@@ -0,0 +1,14 @@
+name: Jibauni
+authors: TheGiraffe3
+homepage: https://github.com/TheGiraffe3/Jibauni
+license: CC-BY-SA-4.0
+version: 0.6
```

```suggestion
version: v0.6
```
use the exact name of your tag here